libs/sprite-scaling/scaling.ts

```diff
@@ -94,17 +88,15 @@ namespace sprites {
         sprite.setScaleCore(sx, sy, anchor, proportional);
     }
 
-    //% blockId=sprite_shrink_by_pixels
+    //% blockId=sprite_shrink_by_pixels_ex
```

Comment:
this is a breaking change. not a problem since it's beta just thought I'd point it out :)

Reply:
> this is a breaking change. not a problem since it's beta, just thought I'd point it out :)

Yep, I would not have made this change if it were released.